test/Microsoft.ML.CodeAnalyzer.Tests/Resources/ContractsCheckResource.cs

```diff
@@ -65,7 +65,17 @@ namespace Microsoft.ML
     internal enum MessageSensitivity
     {
         None = 0,
-        Unknown = ~None
+        Unknown = ~None,
+        UserData,
+        Schema
     }
-    internal interface IHostEnvironment : IExceptionContext { }
+    internal interface IHostEnvironment : IExceptionContext
```

Comment:
new line

Reply:
As you might expect with code meant to test the analyzer I am a bit less picky about code formatting versus actual code. If it's all the same to you I do not intend to "fix" this.